react/Image.tsx

```diff
@@ -8,6 +8,9 @@ import styles from './styles.css'
 export interface ImageProps extends ImgHTMLAttributes<HTMLImageElement> {
   maxWidth?: string | number
   maxHeight?: string | number
+  minWidth?: string | number
+  minHeight?: string | number
+  shouldFillWidth?: boolean
```

Comment:
is this prop introduced by your PR or is it new? if so, I suggest turning it into `fullWidth` for consistency with `flex-layout`, which also has a prop with this name

Reply:
This is introduced by this PR. And it makes sense to name it `fullWidth`, nice!